.github/workflows/size.yml

```diff
@@ -21,5 +21,6 @@ jobs:
         uses: preactjs/compressed-size-action@v2
         with:
           pattern: 'test/fixtures/**/dist/!(*.map)'
-          build-script: 'test'
+          # We don't care about test suite failures, that's for the CI action
+          build-script: 'test -- -u'
```

Comment:
Perhaps questionable, but https://github.com/developit/microbundle/pull/1081/commits/fe9728619c8344252a77e161f20844695ea833bf shows a failing compressed-size action when the real 'Test' action passes just fine.

We shouldn't care about test failures here though.